container/features/src/main/resources/features.xml

```diff
@@ -65,6 +65,9 @@
         <bundle>wrap:mvn:org.apache.xmlgraphics/batik-util/${batikVersion}</bundle>
         <bundle>wrap:mvn:org.apache.xmlgraphics/batik-xml/${batikVersion}</bundle>
     </feature>
+    <feature name="bcprov" version="${bcprovVersion}" description="Legion of the Bouncy Castle :: Java Cryptography APIs">
```

Comment:
We should verify that the WMI monitor/collector load & function properly on Minion with these changes.